### ISSUE TITLE
code

### ISSUE THREAD

**Issue by 1390806607** [KidsWithTokens/MedSegDiff#148]
res = torch.where(mask > 0, 1, 0)   #merge all tumor classes into one to get a binary segmentation mask

Hello, I don't quite understand why all categories of masks are merged into two categories of masks.Can you explain it to me?
